ocp_resources/controller_revision.py

```diff
@@ -0,0 +1,5 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class ControllerRevision(NamespacedResource):
```

Comment:
Please include reference to the resource
https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/controller-revision-v1/

Please implement under to_dict the required resource data.